packages/docs/src/components/sidebar/navigationRoutes.ts

```diff
@@ -59,6 +59,10 @@ export const navigationRoutes: NavigationRoute[] = [
         name: 'ssr-guide',
         displayName: 'menu.ssrGuide',
       },
+      {
+        name: 'kitchensink',
+        displayName: 'menu.kitchensink',
```

Comment:
Affix and backtop are pretty horrible, can we do something about them?

<img width="1112" alt="image" src="https://user-images.githubusercontent.com/5394573/189341946-7c767d9a-32e3-4fb2-8337-d420dfaf2bb9.png">

Reply:
We can create good examples :)

Reply:
@Azgiliat I mean maybe replacing default example (pretty sure both backtop and affix have containerized versions).

Reply:
Backtop excluded, affix is placed in container now